src/libclipper/include/clipper/constants.hpp

```diff
@@ -16,8 +16,6 @@ enum RedisDBTable {
   REDIS_APP_MODEL_LINKS_DB_NUM = 7,
 };
 
-constexpr int RPC_SERVICE_PORT = 7000;
```

Comment:
This was moved to the Conf struct instead right?

Reply:
You are right.